scripts/image

```diff
@@ -228,7 +228,7 @@ for MOD in `find $INSTALL/usr/lib/kernel-overlays/ -type f -name *.ko`; do
 done
 
 # symlink overlayed modules to /usr/lib/modules
-ln -sf /var/lib/modules $INSTALL/usr/lib/modules
+ln -sT /var/lib/modules $INSTALL/usr/lib/modules
```

Comment:
Why not leave f (force)?

Reply:
This is to make sure we really create a symlink named "modules" in $INSTALL/usr/lib/

If some package created a $INSTALL/usr/lib/modules directory by mistake "ln -sf" would create the symlink "modules" in $INSTALL/usr/lib/modules. with -sT build will fail in that case